### ISSUE TITLE
training set in VLM2Vec-v2

### ISSUE THREAD

**Issue by FightingFighting** [TIGER-AI-Lab/VLM2Vec#129]
The VLM2Vec-v2  didnot contain any training sample right?

**Comment by XMHZZ2018** [comment]
@FightingFighting 

We have currently only released the test set & benchmark (MMEB-v2). The video and document training data have not been released yet, and the image training data is the same as in MMEB-v1.

**Comment by XMHZZ2018** [comment]
Sorry, I think I may have made an incorrect claim earlier. As far as I know, all the training data used for the currently released model has been made publicly available.

Based on the training data configuration (https://github.com/TIGER-AI-Lab/VLM2Vec/blob/main/experiments/public/train/train_alltasks.yaml):

1. For image data, we continue to use MMEB-v1-train.

2. For Visdoc data, we use Colpali-train and Visrag-train, both of which are public and can be loaded directly from Hugging Face.

3. For video data, we use LLaVA-Hound, which is also public and available on Hugging Face.

While we do have future model checkpoints that may rely on unreleased data sources, all the training data used for the currently released model checkpoint is publicly accessible.

**Comment by FightingFighting** [comment]
Hi @XMHZZ2018 thank you very much.
